apps/server/src/modules/school/uc/school.uc.ts

```diff
@@ -27,6 +27,18 @@ export class SchoolUc {
 			action: Actions.read,
 			requiredPermissions: [Permission.SCHOOL_EDIT],
 		});
+		const school: SchoolDO = await this.schoolService.getSchoolById(schoolId);
+
+		const shouldRestartMigration =
```

Comment:
Can you please simplify it and move to private methode. Why you need !! ?

Reply:
@IgorCapCoder  please see my comment and think about it

Reply:
I have done it now